provisioning/provisioning-device-client/src/main/java/com/microsoft/azure/sdk/iot/provisioning/device/internal/task/ProvisioningTask.java

```diff
@@ -180,7 +180,6 @@ private void executeStateMachineForStatus(RegistrationOperationStatusParser regi
 
     {
         boolean isContinue = false;
-        RegistrationOperationStatusParser statusRegistrationOperationStatusParser = registrationOperationStatusParser;
```

Comment:
We were copying the parser class here for no reason in particular. I've removed this and we just reference ```registrationOperationStatusParser``` everywhere below now instead